src/zarr/store/zip.py

```diff
@@ -51,18 +52,18 @@ class ZipStore(Store):
 
     def __init__(
         self,
-        path: Path | str,
+        file_path: Path | str,
         *,
+        path: str = "",
```

Comment:
This feels quite confusing. IIUC, you are allowing the caller to specify a path within the zipfile as opposed to putting the root of the store at the root of the zip, is that right?

Reply:
exactly! this is required to make `ZipStore` congruent to `RemoteStore`

Reply:
though this may have some impact (*simplifying*) https://github.com/zarr-developers/zarr-specs/pull/311